sdks/java/core/src/main/java/com/google/cloud/dataflow/sdk/transforms/Filter.java

```diff
@@ -163,6 +175,11 @@ public void processElement(ProcessContext c) {
           c.output(c.element());
         }
       }
+
+      @Override
+      public void populateDisplayData(DisplayData.Builder builder) {
+        Filter.populateDisplayData(builder, String.format("x ≤ %s", value));
```

Comment:
@davorbonaci for comments on using unicode in the codebase. I'm not sure if we should care.